### PR TITLE
MEL-387: Fix edit existing viz small bugs

### PR DESCRIPTION
**Pull request by chris-bes**
### Changes:

- Fix edit existing viz small bugs

### CODE REVIEW

File: packages/admin-panel/src/VizBuilderApp/api/constants.js [hunk 1]
```diff
@@ -7,6 +7,5 @@
 export const DEFAULT_REACT_QUERY_OPTIONS = {
   retry: 0,
   // should be refetched in the background every hour
-  staleTime: 1000 * 60 * 60,
```
Comment (chris-bes) [inline packages/admin-panel/src/VizBuilderApp/api/constants.js:10]: The query cache has caused some annoying scenarios such as when you update the viz from somewhere else (Admin Panel Dashboard Item modal) and it doesn't reflect in the viz-builder page. So I think it's better to disable the cache and always fetch the latest data

File: packages/admin-panel-server/src/viz-builder/dashboardVisualisation/DashboardVisualisationCombiner.ts [hunk 2]
```diff
@@ -34,7 +34,7 @@ export class DashboardVisualisationCombiner {
     const { config: reportConfig } = this.report;
     const { output } = reportConfig;
     const { config: dashboardItemConfig } = this.dashboardItem;
-    const { type, ...config } = dashboardItemConfig;
+    const { type, name, ...config } = dashboardItemConfig;
```
Comment (chris-bes) [inline packages/admin-panel-server/src/viz-builder/dashboardVisualisation/DashboardVisualisationCombiner.ts:37]: The `name` of the viz should be updated by the Main Edit modal of the viz-builder, not through chart config. So this is to excluding it from the chart config

File: packages/admin-panel/src/VizBuilderApp/api/constants.js (hunk 1 above)
Comment (kael89) [reply packages/admin-panel/src/VizBuilderApp/api/constants.js:10]: Good call - viz builder data should be fairly quick to fetch!

File: packages/admin-panel/src/VizBuilderApp/api/queries/useReportPreview.js [hunk 3]
```diff
@@ -20,6 +21,7 @@ export const useReportPreview = (visualisation, project, location, enabled, setE
       return response.results;
     },
     {
+      ...DEFAULT_REACT_QUERY_OPTIONS,
```
Comment (kael89) [inline packages/admin-panel/src/VizBuilderApp/api/queries/useReportPreview.js:24]: A nice improvement would be defining our own `query` helper, which uses `DEFAULT_REACT_QUERY_OPTIONS`

**Review by kael89 — APPROVED** [review]
Looks good @chris-bes !

**Pull request merged**